RusTaxi/Classes/Screens/VC/SlideShow Controller/MainController.swift

```diff
@@ -37,6 +25,12 @@ class MainController: UIViewController, MKMapViewDelegate, CLLocationManagerDele
 		initializeFirstAddressCells()
 	}
 	
+	override func viewWillAppear(_ animated: Bool) {
+		super.viewWillAppear(true)
+		
+		tableView.estimatedRowHeight = 63
```

Comment:
tableView.estimatedRowHeight -> Delegate Function estimatedHeightForRow of `UITableViewDelegate`
also for heightForRow